src/login.js

```diff
@@ -46,27 +48,34 @@ class Login extends Component {
     e.preventDefault();
     const options = {
       method: "POST",
-      uri: this.props.tokenUrl,
-      formData: {
+      url: this.props.tokenUrl,
+      headers: {
+        "content-type":"application/json"
+      },
+      body: {
         grant_type: "password",
         username: this.state.username,
         password: this.state.password,
-        scope: this.props.scope
+        audience: this.props.audience,
+        scope: this.props.scope,
+        client_id: this.props.client_id
       },
-      headers: {
-        "content-type":"application/x-www-form-urlencoded"
-      }
-    }
+      json: true
+    };
+    
     rp(options)
-      .then(token => {
-        localStorage.set("token", token);
+      .then(response => {
+        const token = response.access_token;
+        const token_type = response.token_type;
+        this.changeStatusGetTokenError(false);
+        localStorage.setItem("token", `${token_type} ${token}`);
```

Comment:
Please split this to:
```
const refresh_token = response.refresh_token
localStorage.setItem("access_token", token);
localStorage.setItem("refresh_token", refresh_token);
localStorage.setItem("token_type", token_type);
```

Reply:
response.refresh_token
The response don't have the property.